README.md

```diff
@@ -10,28 +10,192 @@ The features are available before an official release so that users and collabor
 
 ---
 
-TorchRL is an open-source Reinforcement Learning (RL) library for PyTorch. 
+**TorchRL** is an open-source Reinforcement Learning (RL) library for PyTorch. 
 
-It provides pytorch and python-first, low and high level abstractions for RL that are intended to be efficient, documented and properly tested. 
+It provides pytorch and **python-first**, low and high level abstractions for RL that are intended to be **efficient**, **modular**, **documented** and properly **tested**. 
 The code is aimed at supporting research in RL. Most of it is written in python in a highly modular way, such that researchers can easily swap components, transform them or write new ones with little effort.
 
 This repo attempts to align with the existing pytorch ecosystem libraries in that it has a dataset pillar ([torchrl/envs](torchrl/envs)), [transforms](torchrl/envs/transforms), [models](torchrl/modules), data utilities (e.g. collectors and containers), etc. 
 TorchRL aims at having as few dependencies as possible (python standard library, numpy and pytorch). Common environment libraries (e.g. OpenAI gym) are only optional.
 
 On the low-level end, torchrl comes with a set of highly re-usable functionals for [cost functions](torchrl/objectives/costs), [returns](torchrl/objectives/returns) and data processing.
 
-On the high-level end, torchrl provides:
-- multiprocess [data collectors](torchrl/collectors/collectors.py);
-- efficient and generic [replay buffers](torchrl/data/replay_buffers/replay_buffers.py);
-- [TensorDict](torchrl/data/tensordict/tensordict.py), a convenient data structure to pass data from one object to another without friction;
+TorchRL aims at a high modularity (1) and good runtime performance (2).
+
+## Features
+
+On the high-level end, TorchRL provides:
+- [`TensorDict`](torchrl/data/tensordict/tensordict.py), 
+a convenient data structure<sup>(1)</sup> to pass data from 
+one object to another without friction.
+`TensorDict` makes it easy to re-use pieces of code across environments, models and
+algorithms. For instance, here's how to code a rollout in TorchRL:
+    <details>
+      <summary>Code</summary>
+    
+    ```diff
```

Comment:
@msaroufim is this what you had in mind? i.e. showing what the primitives bring via diff code style?